nlp_primitives/utilities.py

```diff
@@ -7,25 +6,25 @@
 
 
 def clean_tokens(textstr):
+    textstr = textstr.translate(str.maketrans('', '', string.punctuation))
```

Comment:
is there a reason to use this instead of just replacing all punctuation with an empty string with regex maybe? This could be a better way/not the only reason for this line (it seems like this line is meant to remove all punctuation, but feel free to correct me), but just a thought/wondering the rationale for this construction in particular. I see that you have removed the `import re` line so a reason could be that regex was really slow?

Reply:
Everything I've found online indicates that this `maketrans` is just about the most efficient way to do this operation, apparently it just directly wraps C code, which is nice. See my comment below about regex and why I moved away from that.

Reply:
oooh cool!! the more ya know!